backend/index.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const { PrismaClient } = require('@prisma/client');
+// const cors = require('cors');
+
+const prisma = new PrismaClient();
+const app = express();
+const PORT = 3000
+
+app.use(express.json());
+
+
+app.get('/', (req, res) => {
+    res.send('Welcome to the API!');
+  });
+
+// route to create new user
+app.post('/users', async (req, res) => {
+    const { firstName, lastName, email, uid, profileURL} = req.body;
+
+    const existingUser = await prisma.user.findUnique({ where: { email } });
+    if (existingUser) {
+        return res.status(400).json({ error: 'Account already in use' });
+     }
+    const newUser = await prisma.user.create({
```

Comment:
Once the user does the login steps through Firebase, you should be able to show them this info.

How how can you query Prisma to access this information after the Firebase login?

More specifically:

What info do you need from Firebase to query for a specific user?

Is the information you're storing here sufficient to let you lookup a user from the info Firebase provides? 

And what info does firebase provide to that might let you identify the user? (e.g. firebase user key)